GETTING_STARTED.md

```diff
@@ -0,0 +1,415 @@
+
+You will also have to [set up the GOPATH](https://github.com/golang/go/wiki/SettingGOPATH) env variable (the instructions are for Go 1.13 but it's the same for Go 1.12).
+
+### Installing on a Mac
+
+All the components (except operator-sdk v0.8.1) can be easily installed via [Homebrew](https://brew.sh/) on a Mac:
+
+```shell
+brew install git
+brew install go@1.12
+brew install docker
+brew install minikube
+brew install kubectl
+```
+
+Unfortunately, operator-sdk in version 0.8.1 isn't available via brew so you have to install it from GitHub release as described [on this page](https://github.com/operator-framework/operator-sdk/blob/v0.8.1/doc/user/install-operator-sdk.md).
+
+### Checkout your fork
+
+You wil need your own Eunomia fork later in the second example. When pushing changes to your fork, you will observe Eunomia make adjustments based on your changed GitOpsConfig.
+
+Check out [the Eunomia repository](https://github.com/KohlsTechnology/eunomia).
+
+1. Create a [fork](https://help.github.com/en/articles/fork-a-repo) of it
+2. Create the directories and clone your fork:
+
+```
+mkdir -p ${GOPATH}/src/github.com/KohlsTechnology
+cd ${GOPATH}/src/github.com/KohlsTechnology
+git clone https://github.com/<YOUR FORK>/eunomia.git
+cd eunomia
+```
+
+## Deploy Eunomia to a Minikube cluster
+
+### Start a local Minikube cluster
+
+Minikube is a minimal Kubernetes cluster run in a virtual machine in Virtualbox.
+
+```shell
+minikube start
+```
+
+From now on your local Kubernetes client (kubectl) is configured to use your just started Minikube cluster.
+
+### Deploy images to Minikube Docker registry
+
+The Eunomia operator gets packaged as a container image - we need to build it (with some other template processor images which [are described here](./README.md#Template-Engine)) and send to Minikube Docker registry to be easily accessible from the cluster.
+
+To achieve this run the helper script (assuming you are in your local Eunomia repository dir):
+
+```shell
+./scripts/deploy-to-local.sh minikube
+```
+
+### Deploy Eunomia to your Minikube cluster
+
+It'll be done with the help of [Helm](https://helm.sh/). If you use a Mac, you can easily install it via Homebrew.
+
+```shell
+helm template deploy/helm/eunomia-operator/ \
+  --set eunomia.operator.image.tag=dev \
+  --set eunomia.operator.image.pullPolicy=Never | kubectl apply -f -
+```
+
+Now you can make sure that Eunomia has been successfully deployed. Just issue:
+
+```shell
+kubectl get pods -n eunomia-operator
+```
+
+The output should be similar to this one:
+
+```shell
+NAME                               READY   STATUS    RESTARTS   AGE
+eunomia-operator-74468f88b-ff9rk   1/1     Running   0          9s
+```
+
+## Hello-world example
+
+We'll be using a very simple yaml files example which you can find [here](./examples/hello-world-yaml/README.md). Additionaly, we'll discuss every single step performed to make sure that everything regarding Eunomia operation is clear.
```

Comment:
please make whole path visible instead of using `here` as link name, so that this guide stays usable when printed on paper or copy-pasted - e.g.:

    you can find in: [examples/hello-world-yaml/README.md](examples/hello-world-yaml/README.md).